src/components/pages.js/BooksContainer.js

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- Kindly remove this commented piece of code, it makes your code not maintainable and clean.